pkg/utils/utils.go

```diff
@@ -53,3 +57,24 @@ const (
 	ContextRequestRemoteAddr = key("request-remote-addr")
 	ContextAuditKey          = key("request-audit-entry")
 )
+
+// GetOperatorRuntime Retrieves the runtime from env variable
+func GetOperatorRuntime() common.Runtime {
+	envString := os.Getenv(common.OperatorRuntimeEnv)
+	runtimeReturn := common.OperatorRuntimeK8s
+	if envString != "" {
+		envString = strings.TrimSpace(envString)
+		envString = strings.ToUpper(envString)
+		switch envString {
```

Comment:
May be we can maintain a `map[string]common.Runtime` and return value as
```
if envString != "" {
	if val, ok := m[envString]; ok {
		runtimeReturn = val
	}
}
```
This way we don't need to keep adding cases later.

Reply:
Sure, is a good Idea, on place already